pandas/tests/util/test_assert_produces_warning.py

```diff
@@ -178,6 +178,14 @@ def test_same_category_different_messages_last_match():
         warnings.warn("Do not match that either", category)
         warnings.warn("Match this", category)
 
+def test_match_multiple_warnings():
```

Comment:
please put the github issue number as a comment here (see some other tests for examples)